openmetadata-ui/src/main/resources/ui/src/constants/docs.constants.ts

```diff
@@ -75,3 +75,18 @@ export const AIRFLOW_DOCS = 'https://docs.open-metadata.org/deployment/airflow';
 
 export const CUSTOM_LOGO_DOCS =
   'https://docs.open-metadata.org/how-to-guides/how-to-add-custom-logo';
+
+export const FOLLOW_DATA_ASSET =
+  'https://docs.open-metadata.org/v1.2.x/how-to-guides/guide-for-data-users/follow-data-asset';
```

Comment:
Remove version from all the links,
We have handled latest version ingestion on the docs side so
https://docs.open-metadata.org/how-to-guides/guide-for-data-users/follow-data-asset
will point to always latest one